tests/aws/services/cloudformation/resources/test_lambda.py

```diff
@@ -305,7 +300,12 @@ def test_event_invoke_config(deploy_cfn_template, snapshot, aws_client):
     snapshot.match("event_invoke_config", event_invoke_config)
 
 
-@markers.snapshot.skip_snapshot_verify(paths=["$..CodeSize"])
+@markers.snapshot.skip_snapshot_verify(
+    paths=[
+        # Lambda ZIP flaky in CI
```

Comment:
@dfangl @dominikschubert Is that assumption correct?

Reply:
Usually yes, we cannot totally rely on this, as it might change.